bemani/utils/read.py

```diff
@@ -2839,15 +2839,20 @@ def __init__(
         no_combine: bool,
         update: bool,
     ) -> None:
-        actual_version = {
-            '1': VersionConstants.MUSECA,
-            '1+1/2': VersionConstants.MUSECA_1_PLUS,
-        }.get(version, -1)
+        if version in ['1', '1+1/2']:
```

Comment:
Why can't this just be a new key in the above dictionary?

Reply:
I wasn't sure how to handle it but will do